dummyserver/handlers.py

```diff
@@ -34,7 +31,20 @@ def __call__(self, request_handler):
         for header,value in self.headers:
             request_handler.add_header(header,value)
 
-        request_handler.write(self.body)
+        # chunked
+        if isinstance(self.body, list):
+            for item in self.body:
+                if not isinstance(item, bytes):
+                    item = item.encode('utf8')
+            request_handler.write(item)
+            request_handler.flush()
+        else:
+            body = self.body
+            if not isinstance(body, bytes):
+                body = body.encode('utf8')
+
+            request_handler.write(body)
+
```

Comment:
Woah, wait. It seems to discard output (but is chunked...) I have to check again...